### ISSUE TITLE
error after installation

### ISSUE THREAD

**Issue by melaxon** [dan-da/subaddress-derive-xmr#2]
`yannislivasov@MacBook-Air-Number-1 subaddress-derive-xmr % ./subaddress-derive-xmr --gen-wallet  -g
`
```
Uncaught Exception. code: 2, message: Undefined property: App\Utils\MyLogger::$_last_time
/Users/yannislivasov/labs/subaddress-derive-xmr/src/Utils/MyLogger.php : 192

Stack Trace:
#0 /Users/yannislivasov/labs/subaddress-derive-xmr/src/Utils/MyLogger.php(192): strictmode\_global_error_handler(2, 'Undefined prope...', '/Users/yannisli...', 192)
#1 /Users/yannislivasov/labs/subaddress-derive-xmr/src/Utils/MyLogger.php(210): App\Utils\MyLogger->format_log_msg('#1: ErrorExcept...', 4, true)
#2 /Users/yannislivasov/labs/subaddress-derive-xmr/src/Utils/MyLogger.php(179): App\Utils\MyLogger->log('#1: ErrorExcept...', 4)
#3 /Users/yannislivasov/labs/subaddress-derive-xmr/subaddress-derive-xmr(110): App\Utils\MyLogger->log_exception(Object(ErrorException))
#4 /Users/yannislivasov/labs/subaddress-derive-xmr/subaddress-derive-xmr(119): main()
#5 {main}

WARNING: strictmode\ALERTS_MAILTO not defined in config.  To disable this warning, define 'strictmode\ALERTS_DISABLE'.  alert not sent with subject: [MacBook-Air-Number-1.local] [subaddress-derive-xmr] Uncaught exception! [pid: 11335]


Now exiting.  Please report this problem to the software author

```
`yannislivasov@MacBook-Air-Number-1 subaddress-derive-xmr % 
`

**Comment by melaxon** [comment]
Actually it is not compatible with PHP 8.xx

**Comment by dan-da** [comment]
thx for the report.    Fixed in 8afb158b74d173c6d452622e9f8c21557caa354f.
